cartographer_ros/cartographer_ros/assets_writer_main.cc

```diff
@@ -150,7 +150,7 @@ void Run(const string& trajectory_filename, const string& bag_filename,
     rosbag::Bag bag;
     bag.open(bag_filename, rosbag::bagmode::Read);
     rosbag::View view(bag);
-    const ::ros::Time begin_time = view.getBeginTime();
+    const ::builtin_interfaces::msg::Time begin_time = view.getBeginTime();
```

Comment:
Yeah, that's fine.  I'll actually revert this back to the original, and then we'll just have to port the whole thing in the future.